environments/staging/inventory.ini.j2

```diff
@@ -41,11 +41,17 @@ rds_pg0
 rds_pgformplayer0
 
 {{ __pgformplayer_nlb__ }}
+{{ __pgmain_nlb__ }}
+{{ __pgucr_nlb__ }}
+{{ __pgsynclogs_nlb__ }}
 
 [postgresql:children]
 pgproxy2
 remote_postgresql
 pgformplayer_nlb
+pgmain_nlb
+pgucr_nlb
+pgsynclogs_nlb
```

Comment:
I think to fix the build you need to add these to `[ansible_skip:children]` as well